src/components/store/Diamonds/index.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from './style.module.scss';
+
+const numberFormat = new Intl.NumberFormat('en-US');
+
+interface DiamondsProps {
+  count: number;
+  className?: string;
+}
+
+const Diamonds = ({ count, className }: DiamondsProps) => {
```

Comment:
```suggestion
const Diamonds = ({ count, className }: DiamondsProps) => <span></span>
```


optional nitpick: immediate function return values can be inlined (even for components).

with proper indentation though, not all on one line